### DISCUSSION TITLE
API Documentation

### DISCUSSION THREAD

**Discussion by 1mfaasj** [spotweb/spotweb#871] [Q&A] [answered]
Hi, is there some kind of documentation what I can do with the spotweb API?
I know that its possible to use the api in Sabnzbd but can I do more with it?
for instance what I'm looking for: retrieve spots (names), trigger new spots

Thanks

**Comment by Sweepr** [comment, marked as answer]
The API is based on this: https://inhies.github.io/Newznab-API/
